src/Resources/web/app_dev.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+use Contao\ManagerBundle\ContaoManager\Plugin as ManagerBundlePlugin;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+// The check is to ensure we don't use .env in production
```

Comment:
> can you guarantee that no bundle does some getenv because ist is already dotenv aware?
> What do you expect users to do when they want to activate it?

There is no *dotenv aware*. Dotenv is to configure your environment, not extensions or bundles. That's what your Symfony config is for, in our case the parameters.yml. The only way an env variable could be used is by placing such a configuration in parameters.yml, and that means you know about environment variables and should use them on your server platform.

There is no case where a **managed edition** has variables it does not know of. If you have that case, you either do not have a managed edition. Or you can always not generate the entry points and adjust them to your needs.